base/linalg/bunchkaufman.jl

```diff
@@ -86,8 +86,8 @@ convert(::Type{BunchKaufman{T}}, B::BunchKaufman) where {T} =
 convert(::Type{Factorization{T}}, B::BunchKaufman{T}) where {T} = B
 convert(::Type{Factorization{T}}, B::BunchKaufman) where {T} = convert(BunchKaufman{T}, B)
 
-size(B::BunchKaufman) = size(B.LD)
-size(B::BunchKaufman, d::Integer) = size(B.LD, d)
+size(B::BunchKaufman) = size(getfield(B, :LD))
+size(B::BunchKaufman, d::Integer) = size(getfield(B, :LD), d)
```

Comment:
Why switch to explicitly calling `getfield` here?

Reply:
To avoid an infinite cycle between `size` and `getproperty`. The reason is that `getproperty` always calls `size`.

Reply:
Cheers, and thanks for the explanation! :)